### PR TITLE
Add logging for divrem tests

### PR DESCRIPTION
**Pull request by danmosemsft**
In the case of a failure in divrem, log the random parameters being operated on. This should make the next hit on https://github.com/dotnet/corefx/issues/7806 reproducible.

@mellinoe

### CODE REVIEW

**Comment by mellinoe** [comment]
Thanks, this should help identify the problem.

File: src/System.Runtime.Numerics/tests/BigInteger/divrem.cs [hunk 1]
```diff
@@ -181,11 +181,19 @@ public static void RunDivRemTests()
...
-                Assert.Equal(sc.snCalc.Peek().ToString(), sc.myCalc.Peek().ToString());
-                sc.VerifyOutParameter();
+                StackCalc sc = new StackCalc(opstring);
+                while (sc.DoNextOperation())
+                {
+                    Assert.Equal(sc.snCalc.Peek().ToString(), sc.myCalc.Peek().ToString());
+                    sc.VerifyOutParameter();
+                }
+            }
+            catch(Exception e) when (!(e is DivideByZeroException))
```
Comment (stephentoub) [inline src/System.Runtime.Numerics/tests/BigInteger/divrem.cs]: Why only when it's not DivideByZeroException?

File: src/System.Runtime.Numerics/tests/BigInteger/divrem.cs [hunk 2]
```diff
@@ -181,11 +181,19 @@ public static void RunDivRemTests()
...
+            {
+                Console.WriteLine($"VerifyDivRemString failed: {opstring} {e.ToString()}");
+                throw;
```
Comment (danmosemsft) [inline src/System.Runtime.Numerics/tests/BigInteger/divrem.cs]: Good idea

File: src/System.Runtime.Numerics/tests/BigInteger/divrem.cs (hunk 1 above)
Comment (danmosemsft) [inline src/System.Runtime.Numerics/tests/BigInteger/divrem.cs]: Fair enough . Too late though :)

**Pull request merged**